src/figuresClasses.test.ts

```diff
@@ -203,7 +201,7 @@ describe('Function "getInfo"', () => {
     const greenCircle = new Circle('green', 13.31);
 
     expect(getInfo(greenCircle))
-      .toBe('A green circle - 556.27');
+      .toBe('A green circle - 556.55');
```

Comment:
The expected information string for a circle with radius 13.31 should be checked for accuracy. Ensure the area calculation is correct and consider using `toBeCloseTo` for floating-point comparisons if necessary.